prodtests/full-system-test/dpl-workflow.sh

```diff
@@ -463,11 +465,11 @@ fi
 (has_detector_reco TOF || has_detector_ctf TOF) && ! has_detector_from_global_reader TOF && add_W o2-tof-reco-workflow "$TOF_CONFIG --input-type $TOF_INPUT --output-type $TOF_OUTPUT $DISABLE_DIGIT_ROOT_INPUT $DISABLE_ROOT_OUTPUT $DISABLE_MC --pipeline $(get_N tof-compressed-decoder TOF RAW 1),$(get_N TOFClusterer TOF REST 1)"
 has_detector_reco ITS && ! has_detector_from_global_reader ITS && add_W o2-its-reco-workflow "--trackerCA $ITS_CONFIG $DISABLE_MC $DISABLE_DIGIT_CLUSTER_INPUT $DISABLE_ROOT_OUTPUT --pipeline $(get_N its-tracker ITS REST 1 ITSTRK)" "$ITS_CONFIG_KEY;$ITSMFT_STROBES;$ITSEXTRAERR"
 has_detector_reco FT0 && ! has_detector_from_global_reader FT0 && add_W o2-ft0-reco-workflow "$DISABLE_DIGIT_ROOT_INPUT $DISABLE_ROOT_OUTPUT $DISABLE_MC --pipeline $(get_N ft0-reconstructor FT0 REST 1)"
-has_detector_reco TRD && (has_detector_matching ITSTPCTRD || has_detector_matching TPCTRD) && ! has_detector_from_global_reader TRD && add_W o2-trd-tracklet-transformer "--disable-irframe-reader $DISABLE_DIGIT_ROOT_INPUT $DISABLE_ROOT_OUTPUT $DISABLE_MC $TRD_FILTER_CONFIG --pipeline $(get_N TRDTRACKLETTRANSFORMER TRD REST 1 TRDTRKTRANS)"
```

Comment:
I commented in the mail thread. If we want to keep the tracklet transformer and TRD tracking separated for async (for example because we have spare CPU resources at step 2 and then save some at step 3 if we just read the calibrated tracklets) then this is fine with me. Otherwise, I would keep it as it is now and add the transformer workflow to step 3

Reply:
ok, thx. As written in the mail thread, if you prefer it is ok for me to write the tracklets in step 3.
I'll not merge this for now, and you can implement it the way you proposed. If you don't find time while in China, also ok, then we can merge this for now, and you can update it later :)

Reply:
fine with me to merge it as is now. I will try to implement a cleaner solution later